kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -2101,6 +2092,32 @@ protected void handleDescribeConfigs(KafkaHeaderAndRequest describeConfigs,
 
     }
 
+    @Override
+    protected void handleDescribeCluster(KafkaHeaderAndRequest describeConfigs,
+                                         CompletableFuture<AbstractResponse> resultFuture) {
+        checkArgument(describeConfigs.getRequest() instanceof DescribeClusterRequest);
+        DescribeClusterResponseData data = new DescribeClusterResponseData();
+        List<Node> allNodes = Collections.synchronizedList(
```

Comment:
Why do we use a synchronized list here?

Reply:
good catch.
it was a left over of a copy/paste